config/webpack.config.js

```diff
@@ -120,6 +120,9 @@ module.exports = env => {
         foremanReact:
           path.join(__dirname,
              '../webpack/assets/javascripts/react_app'),
+        foremanJS:
```

Comment:
Why is this needed?

Reply:
For Katello, so it can use `foreman_toast_notifications.js `

Reply:
We are facing an issue with webpack where `foremanReact` get copied into the build results of each plugin. I afraid after this change `foremanJS` will get copied as well.

IMO it is better to use the `window.tfm.toastNotifications` until the issue will get fixed.

Reply:
Or maybe to leave the small (24 line) `foreman_toast_notifications.js` file in katello?

Reply:
I believe it is better to remove it from katello and use it from the global tfm object so it will be easier to maintain it. Notice plugin developers copy solutions from katello as they are best practices (while they are not). I am afraid to see this code in more plugins.

Reply:
Sure, removed here and changed in the [katello PR](https://github.com/Katello/katello/pull/7965)